test/unit/features/test_pagination_option_validation.py

```diff
@@ -55,3 +55,15 @@ def test_facet_value_errors_for_search(self):
       with self.subTest(invalid_opt):
         with self.assertRaises(ValueError, msg=f'There should be a ValueError for search with option {invalid_opt}.'):
           Pagination.new_pagination(None, PagerType.POST_SEARCH, **{invalid_opt: 'test value'})
+
+  def test_key_value_error_for_docs(self):
+      for pager_type in self.all_doc_paginations:
+        with self.subTest(pager_type):
```

Comment:
Is there a way to check that the error message is having `_DOCS_KEY_ERROR`? Same for the `view_paginations` cases.